mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -440,6 +440,21 @@ func resourceContainerCluster() *schema.Resource {
 										Description: `The Google Cloud Platform Service Account to be used by the node VMs.`,
 									},
 									<% unless version == 'ga' -%>
```

Comment:
Is this beta-only? I see it in v1, but I also don't see the beta API, so maybe I'm missing something

Reply:
Hey, @megan07 Thanks for the review! No, I put that here because while I was writing that, I thought maybe it could be more interesting to add this in the beta first and then put that in the GA. I'm not familiar with the process.

Reply:
@mrvarmazyar - Oh, got it! We try to line up our beta provider with what is in the Google Beta API, so this can be GA. Thanks!

Reply:
@megan07 Thanks, I'll put those changes in GA.